### ISSUE TITLE
broken 'restrict project versioning to root- and sub-projects' 

### ISSUE THREAD

**Issue by dartilla** [qoomon/maven-git-versioning-extension#68]
Having issue with detecting version of multimodule project after 4.10.0 release.

Sample project attached [broken.zip](https://github.com/qoomon/maven-git-versioning-extension/files/4179664/broken.zip)
On root level version (feature-1) detects ok 
```
broken% mvn --non-recursive exec:exec -Dexec.executable='echo' -Dexec.args='${project.version}' -q
feature-1
```
On bundle level version (Feature-1) does not detects. Fallback version "currentVersion" is found
```
broken% cd bundle
broken/bundle% mvn --non-recursive exec:exec -Dexec.executable='echo' -Dexec.args='${project.version}' -q
currentVersion
```

**Comment by qoomon** [comment]
Strange behaviour , but I found the reason already. It does not work if `<version>` tag is not set in pom.xml file
I will try to debug it this evening.

**Comment by qoomon** [comment]
I think I fixed it try version `5.0.1`

**Comment by dartilla** [comment]
Sorry for late answer.
Issue is resolved, thank you.
